run-bot.py

```diff
@@ -87,6 +87,8 @@ def handle(msg):
     if msg.get("text", "").startswith("/"):
         welcome = (
             "Hello!\n"
+            "Any message, picture, sound, video sent to me will be displayed "
+            "live on http://leplatrem.github.io/kinto-telegram-wall/\n
```

Comment:
This should definitely be a config value :)
this bot can be used by anyone with any webpage